artifacts/client/src/index.css

```
@@ -2,6 +2,30 @@
 @import 'tw-animate-css';
 @plugin "@tailwindcss/typography";
 
+@font-face {
+  font-family: 'Barlow';
```

Stylelint (17.14.0)

[error] Expected no quotes around "Barlow" (font-family-name-quotes)

(font-family-name-quotes)